src/index.js

```diff
@@ -20,10 +21,10 @@ class MyServer {
         try {
             this.server = app.listen(this.port);
             await events.once(this.server, 'listening');
-            console.log('Succes start');
+            log('Succes start');
```

Comment:
Sorry for spellchecking, it's better to use the correct grammar in your debug messages. i.e. `Started successfully`